platform/android/AndroidManifest.xml.template

```diff
@@ -10,32 +11,29 @@
                       android:largeScreens="true"
                       android:xlargeScreens="true"/>
 
-    <application android:label="@string/godot_project_name_string" android:icon="@drawable/icon" android:allowBackup="false" $$ADD_APPATTRIBUTE_CHUNKS$$ >
+    <uses-feature android:glEsVersion="0x00020000" android:required="true" />
+
+$$ADD_PERMISSION_CHUNKS$$
+
+    <application android:label="@string/godot_project_name_string" android:icon="@drawable/icon" android:allowBackup="false" tools:ignore="GoogleAppIndexingWarning" $$ADD_APPATTRIBUTE_CHUNKS$$ >
         <activity android:name="org.godotengine.godot.Godot"
                   android:label="@string/godot_project_name_string"
                   android:theme="@android:style/Theme.NoTitleBar.Fullscreen"
                   android:launchMode="singleTask"
                   android:screenOrientation="landscape"
                   android:configChanges="orientation|keyboardHidden|screenSize|smallestScreenSize"
-                  android:resizeableActivity="false">
+                  android:resizeableActivity="false"
+                  tools:ignore="UnusedAttribute">
 
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
-	<service android:name="org.godotengine.godot.GodotDownloaderService" />
-
-
-
+    <service android:name="org.godotengine.godot.GodotDownloaderService" />
 
 $$ADD_APPLICATION_CHUNKS$$
 
     </application>
-    <uses-feature android:glEsVersion="0x00020000" android:required="true" />
-
-$$ADD_PERMISSION_CHUNKS$$
-
-<uses-sdk android:minSdkVersion="18" android:targetSdkVersion="27"/>
```

Comment:
So we no longer define min and target SDK versions? Or is it done differently?

Reply:
Ah I see it's done in `build.gradle` via `platform/android/SCsub`.

Reply:
it should be done in build.gradle not in manifest.